eth/backend.go

```diff
@@ -613,11 +641,10 @@ func (s *Ethereum) Stop() error {
 	if s.seqRPCService != nil {
 		s.seqRPCService.Close()
 	}
+	*/
 	if s.historicalRPCService != nil {
 		s.historicalRPCService.Close()
 	}
-	*/
-
 	// Clean shutdown marker as the last thing before closing db
 	s.shutdownTracker.Stop()
```

Comment:
I think you should stop s.migrator also here

Reply:
added